src/main/cms/cms_menu_ledstrip.c

```diff
@@ -61,7 +61,7 @@ const char * const ledProfileNames[LED_PROFILE_COUNT] = {
     "RACE",
```

Comment:
There are duplicate strings in `lookupTableLEDProfile`. Can we’re not use the same strings?